mmv1/templates/terraform/env_var_context.go.erb

```diff
@@ -1,4 +1,5 @@
-<% example.test_env_vars&.each do |var_name, var_type| -%>
+<% unless example.test_env_vars.nil? -%>
```

Comment:
this replaces the `test_env_vars&` from before. Couldn't figure out how to add `.sort` while still including the `&`. Let me know if you know a better way